cypress/integration/core/map-controls.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { BaseLayer } from '../../../packages/core/src/components/map-base-layers/map-base-layers.interfaces';
+import { Overlay } from '../../../packages/core/src/components/map-overlays/map-overlays.interfaces';
+
+describe('Map Controls', () => {
+  it('should close layer info when layer becomes available', () => {
+    cy.visit('http://localhost:56106/iframe.html?id=map-controls--map-controls');
+
+    cy.get('dso-map-controls').shadow().find('#toggle-visibility-button').click();
+
+    testLayer('dso-map-overlays', 'overlays', 'Riool');
+    testLayer('dso-map-base-layers', 'baseLayers', 'Lavakaart');
+
+    function testLayer<Layer extends Overlay | BaseLayer>(element: 'dso-map-overlays' | 'dso-map-base-layers', property: 'overlays' | 'baseLayers', label: string) {
+      cy.get(element).shadow().contains('dso-selectable', label).shadow().as('selectable').find('dso-info-button').shadow().find('button').click();
+
+      cy.get('@selectable').find('dso-info').should('be.visible')
+
+      // Emulate zoom change
```

Comment:
Hoezo resulteert dit dan in een zoom-change, of hoe 'emuleert' dit dat dan?

Reply:
Dit component is "map framework" agnostisch: Het is niet bekend met zoom levels.

`@dso-toolkit/leaflet-react > @dso-toolkit/leaflet > @dso-toolkit/core`

De package `@dso-toolkit/leaflet` mapt het zoomgedrag naar het model `BaseLayer { disabled: true | false; }`: https://github.com/dso-toolkit/dso-toolkit/blob/master/packages/leaflet/src/map-controls/map-controls.ts#L322

De oneliner op regel 19 in dit PR emuleert dat gedrag. Eigenlijk moet er dus nog een test komen die het gedrag test vanuit `@dso-toolkit/leaflet`. Maar daar zijn we nog niet.